tests/unit/sync/contentTypeHelpers.test.ts

```diff
@@ -0,0 +1,70 @@
+import { describe, expect, it } from "@jest/globals";
+import { ContentTypeElements } from "@kontent-ai/management-sdk";
+
+import { getRequiredIds } from "../../../src/modules/sync/utils/contentTypeHelpers";
+
+const guidelinesRichText =
+  "<p>Don't forget to use the default value. Also, check the default <a data-item-id=\"itemId1\">content Item</a> and check the <a data-item-id=\"itemId2\">showcase item</a></p>\n<figure data-asset-id=\"assetId3\"><img src=\"#\" data-asset-id=\"assetId3\"></figure>";
+
+const assetElements: ContentTypeElements.IAssetElement[] = [
+  {
+    name: "testAsset",
+    type: "asset",
+  },
+  {
+    name: "testAsset1",
+    type: "asset",
+    default: {
+      global: {
+        value: [
+          { id: "assetId1" },
+          { id: "assetId2" },
+        ],
+      },
+    },
+  },
+];
+
+const linkedItemsElements: ContentTypeElements.ILinkedItemsElement[] = [
+  {
+    name: "testAsset",
+    type: "modular_content",
+  },
+  {
+    name: "testAsset1",
+    type: "modular_content",
+    default: {
+      global: {
+        value: [
+          { id: "itemId1" },
+          { id: "itemId2" },
+        ],
+      },
+    },
+  },
+];
+
+const contentType = {
+  id: "id",
+  name: "testType",
+  codename: "test_type",
+  last_modified: "date",
+  elements: [
+    {
+      type: "guidelines",
+      name: "guidelines",
+      guidelines: guidelinesRichText,
+    },
+    ...linkedItemsElements,
+    ...assetElements,
+  ],
+};
+
+describe("getRequiredAssetsIds", () => {
+  it("get asset ids from elements and guidelines rich text", () => {
```

Comment:
The `describe` and `it` descriptions don't match the tested function.

Reply:
I would also prefer to have multiple `it` cases with smaller rich texts for the different cases (item, asset and asset link) instead of one big test case testing all the scenarios.

Reply:
The name in `describe` still doesn't match the name of the tested function.